Felienne/hedy
# [BUG FIX] Admin is able to view own class as teacher again

**Description**
Due to an incorrect comparison the the class overview was always viewed as admin when viewing a class with an admin account. While this might not result in an issue the admin was unable to alter their own class, add students and more because of this limitation. This bug is fixed.

**Fixes**
No related issue.

**How to test**
Make sure you are an admin and a teacher. Make a class and verify that you can indeed add students and customize your class again.
